lib/resque/data_store.rb

```diff
@@ -97,7 +97,7 @@ def server_time
     end
 
     def redis_time_available?
-      @redis.time
+      @redis.time unless Resque.inline
```

Comment:
Yes. From my understanding (and I may be wrong) when `Resque.inline` is set to true, Redis is bypassed entirely and all workers are run in the foreground.